### PR TITLE
Create LaunchDarkly integration docs

### PR DESCRIPTION
**Pull request by masinette**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

*Describe your changes here. If your PR relates to or resolves an issue, add a link to that too.*

## Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: src/docs/product/integrations/index.mdx [hunk 1]
```diff
@@ -105,9 +105,10 @@ For more details, see the [full Integration Platform documentation](/product/int
 
 ## Feature Flag
 
-| Integration                                        | Tie errors to feature flags |
-| -------------------------------------------------- | --------------------------- |
-| [Split](/product/integrations/feature-flag/split/) | X                           |
+| Integration                                                      | Tie errors to feature flags |
+| ---------------------------------------------------------------- | --------------------------- |
+| [Split](/product/integrations/feature-flag/split/)               | X                           |
+| [LaunchDarkly](/product/integrations/feature-flag/launchdarkly/) | X                           |
```
Comment (vivianyentran) [inline src/docs/product/integrations/index.mdx:111]:
```suggestion
| [LaunchDarkly](/product/integrations/feature-flag/launchdarkly/) | X                           |
| [Split](/product/integrations/feature-flag/split/)               | X                           |
```

Switch order to reflect order of pages in navigation

File: src/docs/product/integrations/feature-flag/launchdarkly/index.mdx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+<Note>
+
+Sentry owner or manager permissions are required to install this integration.
+
+</Note>
+
+1. Navigate to **Settings > Integrations > LaunchDarkly**
+
+2. Follow the full [LaunchDarkly installation instructions](https://docs.launchdarkly.com/integrations/sentry).
```
Comment (vivianyentran) [inline src/docs/product/integrations/feature-flag/launchdarkly/index.mdx:22]: Are you sure this is the right link? I get a 404 error

**Review by souredoutlook — APPROVED** [review]

File: src/docs/product/integrations/feature-flag/index.mdx [hunk 3]
```diff
@@ -7,3 +7,4 @@ description: "Learn more about Sentry's feature flag integrations."
 Learn more about Sentry's feature flag integrations:
 
 - [Split](/product/integrations/feature-flag/split/)
+- [LaunchDarkly](/product/integrations/feature-flag/launchdarkly/)
```
Comment (vivianyentran) [inline src/docs/product/integrations/feature-flag/index.mdx:10]:
```suggestion
<PageGrid />
```

Use this to list child pages along with their descriptions.

**Review by vivianyentran — COMMENTED** [review]
Please add a link to the docs on the

File: src/docs/product/integrations/feature-flag/launchdarkly/index.mdx (hunk 2 above)
Comment (souredoutlook) [reply src/docs/product/integrations/feature-flag/launchdarkly/index.mdx:22]: It's not live yet on their end it will be up shortly.

**Pull request merged**